ui/rating-dialog/src/play/java/de/danoeh/antennapod/ui/ratingdialog/RatingDialogFragment.java

```diff
@@ -0,0 +1,49 @@
+package de.danoeh.antennapod.ui.ratingdialog;
+
+import android.app.Dialog;
+import android.os.Bundle;
+import android.text.Html;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.DialogFragment;
+import com.google.android.material.dialog.MaterialAlertDialogBuilder;
+import de.danoeh.antennapod.ui.ratingdialog.databinding.RatingDialogBinding;
+
+public class RatingDialogFragment extends DialogFragment {
+    @Override
+    public void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setCancelable(false);
+    }
+
+    @NonNull
+    @Override
+    public Dialog onCreateDialog(@Nullable Bundle savedInstanceState) {
+        return new MaterialAlertDialogBuilder(getContext())
+                .setView(onCreateView(getLayoutInflater(), null, savedInstanceState))
+                .create();
+    }
+
+    @Nullable
+    @Override
+    public View onCreateView(@NonNull LayoutInflater inflater,
+                             @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
+        RatingDialogBinding viewBinding = RatingDialogBinding.inflate(inflater);
```

Comment:
I think for fragments you need to do `viewBinding = null;` on `onDestroyView`. I know there were concerns about this for other fragments due to EventBus and RxJava, but it should be fine here I think.

https://developer.android.com/topic/libraries/view-binding#fragments

Reply:
This does not apply in this case because the fragment does not keep a reference to the view. The reference is only kept until onCreateView returns.

Reply:
I see 👍